test/e2e/lib/pages/my-home-page.js

```diff
@@ -58,9 +59,14 @@ export default class MyHomePage extends AsyncBaseContainer {
 		return assert( badgeText === 'Complete', 'Could not locate message that email is verified.' );
 	}
 
-	async isSiteLaunched() {
-		await this.closeCelebrateNotice();
-		return await driverHelper.isElementLocated( this.driver, this.launchSiteTaskCompleteLocator );
+	async waitForSiteLaunchComplete() {
+		await driverHelper.waitUntilElementLocated( this.driver, this.celebrateNoticeTitleLocator );
+		const notice = await this.driver.findElement( this.celebrateNoticeTitleLocator );
```

Comment:
Just FYI, after some recent refactors, all the `waitUntilElement*` helpers [return the located element](https://github.com/Automattic/wp-calypso/blob/fa2a08f36c35aa83ba98b2b6f403175db304c983/test/e2e/lib/driver-helper.js#L157-L158). This means we don't need to look for the element again after it was located. The above two lines could be just one line then:
```js
const notice = await driverHelper.waitUntilElementLocated( this.driver, this.celebrateNoticeTitleLocator );
```